src/main/java/frc/robot/commands/mover/TiltedElevatorCommand.java

```diff
@@ -1,16 +1,35 @@
 package frc.robot.commands.mover;
 
-import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
 
 import frc.robot.subsystems.TiltedElevatorSubsystem;
 import frc.robot.subsystems.TiltedElevatorSubsystem.ElevatorState;
 
-public class TiltedElevatorCommand extends InstantCommand {
-    public TiltedElevatorCommand(TiltedElevatorSubsystem tiltedElevatorSubsystem, ElevatorState level) {
-        super(() -> {
-            System.out.println("Mover to " + level);
-            tiltedElevatorSubsystem.setState(level);
-            System.out.println("Mover at " + level);
-        }, tiltedElevatorSubsystem);
+public class TiltedElevatorCommand extends CommandBase {
+    private final TiltedElevatorSubsystem tiltedElevatorSubsystem;
+    private final ElevatorState targetState;
+    private final Timer timer;
+
+    public TiltedElevatorCommand(TiltedElevatorSubsystem tiltedElevatorSubsystem, ElevatorState targetState) {
+        this.tiltedElevatorSubsystem = tiltedElevatorSubsystem;
+        this.targetState = targetState;
+        timer = new Timer();
+
+        addRequirements(tiltedElevatorSubsystem);
+    }
+
+    public void initialize(){
+        System.out.println("Mover to " + targetState);
+        tiltedElevatorSubsystem.setState(targetState);
+        timer.start();
+    }
+
+    public void end(boolean interrupted) {
+        System.out.println("Mover at " + targetState);
+    }
+    
+    public boolean isFinished() {
+        return tiltedElevatorSubsystem.atTarget() || timer.hasElapsed(1);
```

Comment:
do we need to tune this killswitch amt? would it be safer to use `withTimeout` so auto can use a different timeout?

Reply:
I'm going to change this to 3 seconds so that it is never used under normal operation, so it shouldn't affect auton. This way it only uses it if the elevator is taking absurdly long and is actually stuck.